inc/hooks/scripts-styles.php

```diff
@@ -82,16 +86,40 @@ function enqueue_polyfills() {
     return false;
   }
   }());
-  var legacyScript ="' . esc_url( get_theme_file_uri( $legacy_scripts ) ) . '";
+  var legacyScript ="' . esc_url( get_theme_file_uri( get_asset_file( 'legacy', 'js' ) ) ) . '";
   if (!supportsES6) {
     var script = document.createElement("script");
     script.src = legacyScript;
     document.head.appendChild(script);
   }';
 
-  if ( file_exists( get_theme_file_path( $legacy_scripts ) ) ) {
-    wp_register_script( 'air_light_legacy', '', [], filemtime( get_theme_file_path( $legacy_scripts ) ), false );
+  if ( file_exists( get_theme_file_path( get_asset_file( 'legacy', 'js' ) ) ) ) {
+    wp_register_script( 'air_light_legacy', '', [], filemtime( get_theme_file_path( get_asset_file( 'legacy', 'js' ) ) ), false );
     wp_enqueue_script( 'air_light_legacy' );
     wp_add_inline_script( 'air_light_legacy', $script, true );
   }
 }
+
+/**
+ * Returns the built asset filename and path depending on
+ * current environment.
+ *
+ * @param string $name File name without file extension
+ * @param string $type File type (as in extension)
+ * @return string file and path of the asset file
+ */
+function get_asset_file( $name, $type ) {
+  $is_dev = 'development' === getenv( 'WP_ENV' ) && ! isset( $_GET['load_production_builds'] );
```

Comment:
We should actually use the new `wp_get_environment_type` function here.

I just updated the dudestack to support it and change the env based on `WP_ENV`. https://github.com/digitoimistodude/dudestack/commit/f832c884ab92d815ba13de780b8e6604bf6013ad